pnpm-lock.yaml

```diff
@@ -7053,6 +7049,259 @@ packages:
     requiresBuild: true
     optional: true
 
+  /@comet/admin-color-picker@6.3.0(@apollo/client@3.7.4)(@emotion/react@11.9.3)(@emotion/styled@11.10.5)(@mui/icons-material@5.11.0)(@mui/material@5.11.6)(@mui/styles@5.11.2)(@mui/x-data-grid@5.17.20)(@types/react@17.0.53)(final-form@4.20.9)(graphql@15.8.0)(history@4.10.1)(react-dnd@16.0.1)(react-dom@17.0.2)(react-final-form@6.5.9)(react-intl@5.25.1)(react-router-dom@5.3.4)(react-router@5.3.4)(react@17.0.2):
```

Comment:
@nsams something isn't right here: This installs Comet v6.3.0 instead of using the workspace version. Possibly related to #1763